sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/dof/node/VariableMap.scala

```diff
@@ -0,0 +1,58 @@
+package net.sansa_stack.query.spark.dof.node
+
+import org.apache.jena.graph.Node
+import org.apache.jena.sparql.core.Var
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import scala.reflect.ClassTag
+
+class VariableMap[N: ClassTag](spark: SparkContext, constraints: Constraints) extends Serializable {
```

Comment:
Use `SparkSession` instead. Since Spark v.2.x session is unified across all contexts so, please use SparkSession. We in SANSA also use SparkSession.

Reply:
done